pkg/rke/k8s_rke_system_images.go

```diff
@@ -9034,7 +9078,51 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.10",
 		},
-		// Enabled in Rancher v2.7.4
+		// Enabled out of band post v2.7.5
+		"v1.25.11-rancher1-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.5.4",
```

Comment:
`rancher/mirrored-coreos-etcd:v3.5.6` is used in `v1.25.9-rancher2-2`, any reason for going back to an older tag?

Reply:
There was a misunderstanding while updating the June patches. The version was supposed to be downgraded for 1.24. I have now raised a PR to rectify this in the June patches custom branch.